root_profiler/engine/sheet_globals.php

```diff
@@ -7,18 +7,24 @@
     return;
   define ('_SHEET_GLOBALS_INCLUDED_', true);
 
+  function kludgedLookupInData($name)
+  {
+      // Kludge-around: "Weapon" + i + "Crit" used to have a stray 1 between i and Crit.
+      if (!isset($DATA[$name]) && substr($name, 0, 6) == "Weapon" && substr($name, 7) == "Crit")
+          return $DATA[$name.substr(0, 7) + "1Crit"];
+      return $DATA[$name];
+  }
+
   // Echoes the name/value attributes for a tag, in addition to a readonly
   // atribute if necessary.
   function getnv($name, $setread = true)
   {
     global $DATA, $READONLY;
 
-    echo "name=\"$name\" value=\"" . $DATA[$name] . "\"";
-    echo ' title="' . $DATA[$name] . '"';
+    $value = kludgedLookupInData($name);
+    echo 'name="$name" value="$value" title="$value"';
```

Comment:
ah - oops -  PHP is like Unix shells, not Python - single-quoted strings suppress variable expansion.